### PR TITLE
Styling bookstore

### PR DESCRIPTION
**Pull request by xtrahuman**
# styling of bookstore
- [x] bookstore was styled following the details in zeplin

### CODE REVIEW

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
```markdown
Hi Tochukwu, 
While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a professional README. A professional README should have a project title, project description, and instructions to set the project up locally. You may use [this](https://github.com/microverseinc/readme-template) README template from Microverse.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

Invalid Code Review Request **does not count** into the code reviews limit._
```

File: src/components/Books.js [hunk 1]
```diff
@@ -16,11 +16,29 @@ const Books = (props) => {
...
-        <p>{bookType}</p>
-        <p>{name}</p>
+      {/* <div className="allbook-style"> */}
+      <div className="book-detail">
+        <div className="book-type-name">
+          <p className="book-type">{bookType}</p>
+          <p className="book-name">{name}</p>
+        </div>
+        <button type="button" className="remove-button" onClick={() => removeBook(book)}>remove</button>
       </div>
```
Comment (DeliceLydia) [inline src/components/Books.js]: - [ ] Features that haven’t been implemented yet should be added as a placeholder and does not need to be implemented please add  what is missing
ms

File: src/components/Books.js [hunk 2]
```diff
@@ -16,11 +16,29 @@ const Books = (props) => {
 
   return (
     <li>
-      <div>
-        <p>{bookType}</p>
-        <p>{name}</p>
+      {/* <div className="allbook-style"> */}
```
Comment (DeliceLydia) [inline src/components/Books.js]: - [x] Remember to remove commented codes

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @xtrahuman, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

- [ ] Please keep just `README.md` you don't need to duplicate it
- [ ] Fix your live demo link

### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by omar-labana** [comment]
Hi @xtrahuman, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
- [x] As the previous requested, the vertical separator `|` between interaction buttons are still missing. Kindly add them.
- [x] _[OPTIONAL]_ The vertical separator `|` between the book `progress` and chapter `info` should be added to match the design.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by omar-labana** [comment]
Hi @xtrahuman, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by xtrahuman** [comment]
> Hi @xtrahuman,
> 
> Great work on making the changes requested by a previous reviewer 👏 You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
> 
> ## Required Changes ♻️
> * [x]  As the previous requested, the vertical separator `|` between interaction buttons are still missing. Kindly add them.
> * [x]  _[OPTIONAL]_ The vertical separator `|` between the book `progress` and chapter `info` should be added to match the design.
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**